fms2_io/netcdf_io.F90

```diff
@@ -149,6 +150,11 @@ module netcdf_io_mod
   character (len=20) :: time_name
   type(dimension_information) :: bc_dimensions !<information about the current dimensions for regional
                                                !! restart variables
+  logical :: use_collective = .false. !< Flag indicating if we should open the file for collective input
+                                      !! this should be set to .true. in the user application if they want
+                                      !! collective reads (put before open_file())
+  integer :: TileComm=MPI_COMM_NULL   !< MPI communicator used for collective reads.
```

Comment:
Use snake case for variable names.  `Tile_Comm`

Reply:
TileComm -> tile_comm in fms2_io/netcdf_io.F90